site/content/docs/5.1/components/offcanvas.md

```diff
@@ -266,6 +288,7 @@ Bootstrap's offcanvas class exposes a few events for hooking into offcanvas func
 | `shown.bs.offcanvas` | This event is fired when an offcanvas element has been made visible to the user (will wait for CSS transitions to complete). |
 | `hide.bs.offcanvas` | This event is fired immediately when the `hide` method has been called. |
 | `hidden.bs.offcanvas` | This event is fired when an offcanvas element has been hidden from the user (will wait for CSS transitions to complete). |
+| `hidePrevented.bs.offcanvas` | This event is fired when the offcanvas is shown, its backdrop is `static` and a click outside the offcanvas or an escape key press  is performed with the keyboard option or `data-bs-keyboard` set to `false`. |
```

Comment:
```
... `static`, and a click outside of the ...
```

there''s a stray extra space after `press`

that last bit of sentence was very confusing though - the various and and multiple or clauses make it quite unclear what is and isn't mutually exclusive here. worth revisiting/clarifying a bit more? do those or clauses apply just to the "a click outside of the offcanvas or an exscape key press is performed" ?